eng/common/pipelines/templates/steps/credscan.yml

```
@@ -0,0 +1,39 @@
+parameters:
+  SuppressionFilePath: 'eng\CredScanSuppression.json'
```

Use forward slashes to be cross-OS compatible.

```suggestion
  SuppressionFilePath: 'eng/CredScanSuppression.json'
```